TestFoundation/TestNSBundle.swift

```diff
@@ -152,4 +153,13 @@ class TestNSBundle : XCTestCase {
         
         _cleanupPlayground(playground)
     }
+    func test_bundleLoad(){
+        let bundle = NSBundle.main()
+        bundle.load()
+        XCTAssertTrue(bundle.isLoaded)
+        let descriptionString="NSBundle <\(bundle.bundlePath)> (\(bundle.isLoaded  ? "loaded" : "not yet loaded"))"
```

Comment:
I'm going to merge this, but for future reference I don't think it's important to test the output of the debugDescription. No one is allowed to depend on it anyway.